apps/rpe/draft-store-service/sbox.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: draft-store-service
+spec:
+  values:
+    java:
+      disableTraefikTls: true
```

Comment:
no harm to set this globally, i.e. i would add a patch for it and anyway you haven't referenced it so this is doing nothing